pages/common/cowsay.md

```diff
@@ -1,24 +1,28 @@
 # cowsay
 
-> Generate an ASCII character (by default a cow) saying or thinking something.
+> Print an ASCII friend (by default a cow) saying or thinking something.
```

Comment:
Why `friend` instead of `character`?

Reply:
since later on I have used "character" to signify letters rather than the ASCII friends.

Also the -f to me means friend, which is a little easier to remember for users rather than "cow-file"

Reply:
yeah, the source code refers to those as `cowfile` and they are inside the https://github.com/tnalpgge/rank-amateur-cowsay/tree/master/cows folder. :cow:

Reply:
I don't particularly mind this change.  "character" (_especially_ when combined with "ASCII" 🙂) frequently is thought of as a text character, such as 'A' or '>'

Reply:
A more maintained version of the source is here: https://github.com/cowsay-org/cowsay

Reply:
2>1, should I resolve this?

Reply:
@navarroaxel your opinion?

Reply:
Actually I don't like friend, I prefer character.

Reply:
Perhaps we could use `animal` or `creature`?

Reply:
I wonder if @sbrl has any opinions?

Reply:
What about `artwork`?

Reply:
`friend` sound to interaction, `artwork` static, I like it.

Reply:
I feel art is better instead of artwork

Reply:
I will change this now
I will use art.

Reply:
We cannot use animal or creature since some things are animals/creatures E.G. this block of cheese

```
 ________________________
< You can't use animals! >
 ------------------------
   \
    \
      _____   _________
     /     \_/         |
    |                 ||
    |                 ||
   |    ###\  /###   | |
   |     0  \/  0    | |
  /|                 | |
 / |        <        |\ \
| /|                 | | |
| |     \_______/   |  | |
| |                 | / /
/||                 /|||
   ----------------|
        | |    | |
        ***    ***
       /___\  /___\
```

Reply:
Cowsay, isn't meant to be a serious program, its not like dd(disk destroyer), its light-hearted, and meant to just be used for fun, if someone is using cowsay, they are having fun or just bored. They aren't probably going to care if they see friend, rather than artwork. When a program is stored in /usr/bin/games, you know its a game/joke program. I don't see why it has to be so serious 

However axel does have a point point about it being interactive using the word friend, I see where he is coming from, however just from the description, someone can work out that cowsay isn't meant to be a robot friend or whatever

Reply:
What are we going to use?

Reply:
You raise good points, @CleanMachine1.  Why not go with `art`?

Reply:
Look at this, the first line 

> Print an ASCII friend

How can you change that to art without making it sound stupid

Reply:
Remove the article.

>Print ASCII art.

Reply:
Yes thats the answer, pretty obvious but it makes the rest sound wrong as well

> Print ASCII art (by default a cow) saying or thinking something.

Friend fits that more

Reply:
I don't think it sounds that bad, actually.  If cheese can talk, so can art 🙂

Reply:
Make proposed commit and then we will use art. I would like to get rid of this branch since its the last of my fork meaning I can delete it and clone the upstream

Reply:
```suggestion
> Print ASCII art (by default a cow) saying or thinking something.
```

Reply:
Done, @CleanMachine1.